dbms/src/Storages/KVStore/MultiRaft/RegionData.cpp

```diff
@@ -129,6 +143,13 @@ RegionData::WriteCFIter RegionData::removeDataByWriteIt(const WriteCFIter & writ
             map.erase(data_it);
             reportDealloc(delta);
         }
+        else if unlikely (large_default_cf.hasTxn(decoded_val.prewrite_ts))
+        {
+            auto delta = large_default_cf.getTiKVKeyValueSize(pk, ts);
```

Comment:
Maybe its better that we introduce a two-level iterator?
```
struct Iterator {
    Map::Iterator level1;
    Inner::Map::Iterator level2;
};
```
Then, if we want to erase an element, we have to impl like the following
```
iter->level1.erase(iter->level2);
```
However, it could be kind of ugly.